### PR TITLE
Adjust automation identifiers as requested

### PR DESCRIPTION
**Pull request by typfel**
## What's new in this PR?

Adjust automation identifiers as requested

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/GroupDetails/GroupDetailsParticipantCell.swift [hunk 1]
```diff
@@ -91,7 +91,6 @@ class GroupDetailsParticipantCell: UICollectionViewCell, Themeable {
         
         titleLabel.translatesAutoresizingMaskIntoConstraints = false
         titleLabel.font = FontSpec.init(.normal, .light).font!
-        titleLabel.accessibilityIdentifier = "label.groupdetails.username"
```
Comment (mykola-mokhnach) [inline Wire-iOS/Sources/UserInterface/GroupDetails/GroupDetailsParticipantCell.swift]: these labels are also useful for us. Also, it is useful to have a separate id for unique username underneath the name

File: Wire-iOS/Sources/UserInterface/ConnectRequests/IncomingConnectionView.swift [hunk 2]
```diff
@@ -90,6 +90,7 @@ public final class IncomingConnectionView: UIView {
         )
         
         usernameLabel.attributedText = viewModel.title
+        usernameLabel.accessibilityIdentifier = "name"
```
Comment (mykola-mokhnach) [inline Wire-iOS/Sources/UserInterface/ConnectRequests/IncomingConnectionView.swift]: do we also have this id set for outgoing connection view?

Comment (typfel) [reply Wire-iOS/Sources/UserInterface/ConnectRequests/IncomingConnectionView.swift]: which outgoing connection view are you referring to? The profile screen?

File: Wire-iOS/Sources/UserInterface/GroupDetails/GroupDetailsParticipantCell.swift (hunk 1 above)
Comment (typfel) [reply Wire-iOS/Sources/UserInterface/GroupDetails/GroupDetailsParticipantCell.swift]: So I put I back with id = `username`?

Comment (typfel) [reply Wire-iOS/Sources/UserInterface/GroupDetails/GroupDetailsParticipantCell.swift]: and add `username` for the handle

**Pull request merged**